app/models/company.rb

```diff
@@ -0,0 +1,10 @@
+class Company < ApplicationRecord
+  belongs_to :user 
+  validates :name, presence: true
+  validates :website, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true
+  validates :notes, presence: true
```

Comment:
Have you considered making notes optional? Right now, this would make them mandatory.

Reply:
That is a good suggestion. I will make that change now.